assets/js/googlesitekit-api.js

```diff
@@ -7,4 +7,8 @@ if ( typeof global.googlesitekit === 'undefined' ) {
 	global.googlesitekit = {};
 }
 
-global.googlesitekit.api = API;
+if ( typeof global.googlesitekit.api === 'undefined' ) {
+	global.googlesitekit.api = API;
```

Comment:
We should manually create this `API` object above this clause, from only the whitelisted functions we really want to have in there for 3P usage. Right now, this doesn't make any difference, but it keeps us on the safe side.